adoc/quickstart3_chap_suma_keys_and_first_client.adoc

```diff
@@ -41,12 +41,14 @@ For more information, see <<ref.webui.systems.activ-keys>>.
 
 . In the [guimenu]``Base Channels`` drop-down box, select the {sle} channel that you added during
 ifdef::env-github,backend-html5[]
-<<quickstart3_chap_sumasetup_with_yast.adoc#gs-syncing-with-scc, Synchronizing with SCC>>
+<<quickstart3_chap_sumasetup_with_yast.adoc#gs-syncing-with-scc, Synchronizing with SCC>>.
 endif::[]
 ifndef::env-github,backend-html5[]
-<<proc-quickstart-first-channel-sync#proc-quickstart-first-channel-sync, First Channel Sync>>
+<<proc-quickstart-first-channel-sync#proc-quickstart-first-channel-sync, First Channel Sync>>.
 endif::[]
 
+. When the base channel is selected the list of available child channels will get fetched and displayed in real time below the base channel.
```

Comment:
When the base channel is selected, the list of available child channels is fetched and displayed in real time." (Add a comma, remove 'get', delete 'below the base channel').